src/navigation/Router.tsx

```diff
@@ -19,24 +20,25 @@ export enum Routes {
 
 interface RouterProps {
   isAuth: boolean;
-  theme: string;
 }
 
-const router: React.FC<RouterProps> = props => (
-  <NavigationContainer
-    theme={props.theme === Themes.dark ? darkTheme : lightTheme}>
-    <Stack.Navigator
-      initialRouteName={props.isAuth ? Routes.Main : Routes.Login}
-      screenOptions={{headerShown: false}}>
-      <Stack.Screen name={Routes.Login} component={LoginStack} />
-      <Stack.Screen name={Routes.Main} component={MainTabs} />
-    </Stack.Navigator>
-  </NavigationContainer>
-);
+const router: React.FC<RouterProps> = props => {
+  const {theme} = useContext(ThemeContext);
```

Comment:
We should have HOC probably, that will allow us to pass props (theme) in component without import ThemeContext in all files.